### PR TITLE
ECL-808-ECL-807: Corrected multiple issues with labels and legends

### PR DESCRIPTION
**Pull request by TheDanielDoyle**

### CODE REVIEW

File: app/uk/gov/hmrc/economiccrimelevyregistration/views/LiabilityDateView.scala.html [hunk 1]
```diff
@@ -32,12 +33,12 @@
         @errorSummary(ErrorSummaryViewModel(form))
     }
 
-    <h1 class="govuk-heading-l">@messages("liability.date.h1")</h1>
```
Comment (paddynski-opencast) [inline app/uk/gov/hmrc/economiccrimelevyregistration/views/LiabilityDateView.scala.html:35]: Why did you remove this?

File: app/uk/gov/hmrc/economiccrimelevyregistration/views/UtrTypeView.scala.html [hunk 2]
```diff
@@ -34,16 +34,12 @@
             @errorSummary(ErrorSummaryViewModel(form, errorLinkOverrides = Map("value" -> "value_0")))
         }
 
-        <h1 class="govuk-heading-l">@messages("utrType.heading")</h1>
-
-        <p class="govuk-body">@messages("utrType.p1")</p>
-
```
Comment (paddynski-opencast) [inline app/uk/gov/hmrc/economiccrimelevyregistration/views/UtrTypeView.scala.html:40]: Why did you remove those?

File: app/uk/gov/hmrc/economiccrimelevyregistration/views/contacts/AddAnotherContactView.scala.html [hunk 3]
```diff
@@ -37,15 +37,11 @@
         @errorSummary(ErrorSummaryViewModel(form, errorLinkOverrides = Map("value" -> "yes")))
     }
 
-    <h1 class="govuk-heading-l">@messages("addAnotherContact.heading")</h1>
-
-    <p class="govuk-body">@messages("addAnotherContact.legend")</p>
-
```
Comment (paddynski-opencast) [inline app/uk/gov/hmrc/economiccrimelevyregistration/views/contacts/AddAnotherContactView.scala.html:43]: Why did you remove those?

File: app/uk/gov/hmrc/economiccrimelevyregistration/views/LiabilityDateView.scala.html (hunk 1 above)
Comment (TheDanielDoyle) [reply app/uk/gov/hmrc/economiccrimelevyregistration/views/LiabilityDateView.scala.html:35]: Because it shouldn't be there. When you set a legend to asPageHeading, it creates the H1 for you.

File: app/uk/gov/hmrc/economiccrimelevyregistration/views/UtrTypeView.scala.html (hunk 2 above)
Comment (TheDanielDoyle) [reply app/uk/gov/hmrc/economiccrimelevyregistration/views/UtrTypeView.scala.html:40]: Because it shouldn't be there. When you set a legend to asPageHeading, it creates the H1 for you.

In addition, hinttext takes the place of that paragraph.

File: app/uk/gov/hmrc/economiccrimelevyregistration/views/contacts/AddAnotherContactView.scala.html (hunk 3 above)
Comment (TheDanielDoyle) [reply app/uk/gov/hmrc/economiccrimelevyregistration/views/contacts/AddAnotherContactView.scala.html:43]: Because it shouldn't be there. When you set a legend to asPageHeading, it creates the H1 for you.

In addition, hinttext takes the place of that paragraph.

**Review by paddynski-opencast — APPROVED** [review]
